packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -659,14 +664,33 @@ export class ApexDebug extends LoggingDebugSession {
     }
 
     try {
-      this.orgInfo = await new ForceOrgDisplay().getOrgInfo(args.sfdxProject);
-      this.myRequestService.instanceUrl = this.orgInfo.instanceUrl;
-      this.myRequestService.accessToken = this.orgInfo.accessToken;
+      if (args.connectType === CONNECT_TYPE_ISV_DEBUGGER) {
```

Comment:
Is this property something they add manually in their launch.json? Can it be automated in `ApexDebuggerConfigurationProvider`? That class is in `index.ts` of salesforcedx-vscode-apex-debugger

Reply:
Okay, I see you've created a story for this.